packages/sdk-react/src/components/onboarding/hooks/useOnboardingPerson.ts

```diff
@@ -77,15 +78,19 @@ export function useOnboardingPerson(): OnboardingPersonReturnType {
     isPending: isCreateLoading,
     error: createPersonError,
   } = api.persons.postPersons.useMutation(undefined, {
-    onError: () => {},
+    onError: (error) => {
+      showErrorToast(error);
```

Comment:
We have a special function for that: `getAPIErrorMessage(...)`.

Reply:
@radist2s getAPIErrorMessage shows every message from backend. We use new function to show toasts only in cases when we have error.message. In cases when we have error.details we show errors in fields. Initially I set this function empty `onError: () => {}` to not to show any toasts, but we need this toasts for common errors

Reply:
Sorry, thanks, didn't realise that from the start. Can I ask you to add a comment to the function to make it clear what the exception is for?

Reply:
@radist2s yes, thanks. Added description to showErrorToast.ts